### DISCUSSION TITLE
Query Count Change of fields

### DISCUSSION THREAD

**Discussion by workgroupengineering** [codenotary/immudb#1614] [Q&A] [answered]
Considering https://demo.immudb.io/query

I would like to count how many times the customers active field has changed

It can be done?

**Comment by jeroiraz** [comment, marked as answer]
> Considering https://demo.immudb.io/query
> 
> I would like to count how many times the customers active field has changed
> 
> It can be done?

it's not yet implemented in SQL layer but it will be included in the next release, the date is still to be defined
